mapie/multi_label_classification.py

```diff
@@ -217,32 +246,45 @@ def _check_delta(self, delta: Optional[float]):
         Warning
             If delta is not ``None`` and method is CRC
         """
-        if (self.method == "rcps") and (delta is None):
-            raise ValueError(
-                "Invalid delta. "
-                "delta cannot be ``None`` when using "
-                "RCPS method."
-            )
         if (not isinstance(delta, float)) and (delta is not None):
             raise ValueError(
                 "Invalid delta. "
                 f"delta must be a float, not a {type(delta)}"
             )
-        if (delta is not None) and (
-            ((delta <= 0) or (delta >= 1)) and
-            (self.method == "rcps")
-        ):
-            raise ValueError(
-                "Invalid delta. "
-                "delta must be in ]0, 1["
-            )
+        if (self.method == "rcps") or (self.method == "ltt"):
+            if delta is None:
+                raise ValueError(
+                    "Invalid delta. "
+                    "delta cannot be ``None`` when using "
+                    "RCPS or LTT method."
```

Comment:
for the user, it might be clearer to say : 
"delta cannot be ``None`` when controlling "
"recall with RCPS or precision with LTT"
As a user who control the precision might never use the method argument, hence he won't understand this error

Reply:
Done